src/test/java/org/cryptomator/common/settings/SettingsTest.java

```diff
@@ -24,16 +24,12 @@ public void testAutoSave() {
 		Mockito.verify(changeListener, Mockito.times(0)).accept(settings);
 
 		// first change (to property):
-		settings.port.set(42428);
+		settings.directories.add(vaultSettings);
 		Mockito.verify(changeListener, Mockito.times(1)).accept(settings);
 
 		// second change (to list):
-		settings.directories.add(vaultSettings);
-		Mockito.verify(changeListener, Mockito.times(2)).accept(settings);
-
-		// third change (to property of list item):
 		vaultSettings.displayName.set("asd");
-		Mockito.verify(changeListener, Mockito.times(3)).accept(settings);
+		Mockito.verify(changeListener, Mockito.times(2)).accept(settings);
```

Comment:
This test is now wrong. Please stick to the pattern:
1. first change a simple porperty in the settings (e.g. windowsXPosition)
2. Then change a list property (the vault list)
3. then, in a third change, change the property of a list item.